PoW-vs-Pos/en/en_transcription.md

```diff
@@ -0,0 +1,102 @@
+
+(05:24) out longer term the kind of the direction that we're going i think in proof of work uh assuming these systems continue to be around for for quite a while is that you'll see increasing integration between miners and the energy producers themselves we already see for example early signs of co-location where you have a you know an energy producer it could be solar could be hydro could be whatever and they have bitcoin miners on site you know they often will contract with a miner to co-locate them but they
+
+(05:52) could easily do it in a vertically integrated model where the way that the grid works this is something that a lot of kind of environmental criticisms of proof-of-work don't understand is that the electrical grid naturally has to overproduce electricity right otherwise if we didn't have extra uh productive capacity then turning on the marginal laptop would cause a brown out right so we have to have this this spare production always going into the network and then we have to have a way to drain out excess power that's being produced
+
+(06:21) but not consumed and it's actually this really kind of tricky balancing act and so proof of work systems are actually a really good load balancer where if you're you know the type of energy source that is doesn't have full control over your production maybe you're a wind farm maybe you're a hydro dam maybe you're solar or maybe you you have just kind of your your nuclear plant that just kind of has like continuous power but maybe the demand is fluctuating uh throughout the day right so it's more
+
+(06:46) more demand at evening less less demand other times and so i think over time we're going to see that the cheapest source of electricity is that virtually zero cost stranded energy and the best ones to harness that are the producers or people that contract directly with the producers and go on site and so i think over time that's what basically makes it is as kind of diversified as possible and right now i think you know last i checked the biggest bitcoin mining facility in the world is like one percent of total hash rate it's already
+
+(07:17) kind of a structurally decentralized system and i think it only gets more so as you increasingly incorporate that into production since its inception the bitcoin network has operated using proof of work with no plans to change the ethereum network also operates with proof of work but it has future plans to switch to a proof of stake system but what is proof of stake proof of stake is a system where cryptocurrency holders stake their coins use them to vote on the valid blockchain and get rewarded with more coins for
```

Comment:
Distinguere le frasi non del narratore `i think actually when we look out longer term the kind of the direction that we're going i think in proof of work [...] kind of a structurally decentralized system and i think it only gets more so as you increasingly incorporate that into production`